Umbraco-Heartcore/Getting-Started-Cloud/GraphQL-Playground/index.md

```diff
@@ -0,0 +1,93 @@
+---
+versionFrom: 8.0.0
+meta.Title: "Umbraco Heartcore GraphQL Playground"
+meta.Description: "Documentation for Umbraco Heartcore GraphQL Playground"
+---
+
+# GraphQL Playground
+
+With GraphQL query language, you will be able to limit the amount of data transferred to and from your Heartcore instance by specifying exactly what kind of information you would like to get in your API calls.
+
+You can test and build GraphQL queries directly in the backoffice-integrated GraphQL Playground IDE.
+
+In the Settings section in the Umbraco Backoffice, you will find the Headless tree. From there you can use the GraphQL playground to test your queries against your project's schema.
+
+The Playground comes with basic features such as real-time error highlighting, syntax highlighting, formatting, query history and more.
+
+## Accessing GraphQL Playground
+
+1. Log into the project backoffice
+2. Navigate to the Settings dashboard
+3. Find the Headless tree near the bottom of the navigation list
+4. Click on GraphQL Playground to open the IDE
+
+![Where to find GraphQL Playground](images/where-to-find-graphql-playground.png)
+
+From there, you can start querying as soon as you have some content set up!
+
+# Features
+
+## Documentation
+
+GraphQL Playground's built-in documentation can be found on the right side of the IDE, under the "Docs" tab.
+
+The documentation tab grants a quick overview of how content is structured using the document types in your solution. It also gives an idea on how to access specific properties each document type has, as well as their properties.
+
+![Documentation Tab in the Playground](images/docs-helper.png)
+
+The documentation will be automatically updated as you add new document types to match the schema you have on your project.
+
+## Schema
+
+The Schema tab also can be found on the right side of the IDE. It is located below the "Docs" tab.
```

Comment:
```suggestion
The Schema tab can also be found on the right side of the IDE. It is located below the "Docs" tab.
```

Reply:
That was my first though as well, however I decided to move the **also** before **can** - since in the context of this section, I want to say "The schema tab, like the Docs tab, can ALSO be found on the right side". Saying "The Schema tab can also be found on the right side" sounds like that tab can be found in one other location.